.golangci.next.reference.yml

```diff
@@ -2810,10 +2785,35 @@ issues:
   # Default: true
   exclude-use-default: false
 
-  # If set to true exclude and exclude-rules regular expressions become case-sensitive.
+  # If set to true, `exclude` and `exclude-rules` regular expressions become case-sensitive.
   # Default: false
   exclude-case-sensitive: false
 
+  # Which dirs to exclude: issues from them won't be reported.
+  # Can use regexp here: `generated.*`, regexp is applied on full path,
+  # including the path prefix if one is set.
+  # Default dirs are skipped independently of this option's value (see exclude-dirs-use-default).
+  # "/" will be replaced by current OS file path separator to properly work on Windows.
+  # Default: []
+  exclude-dirs:
```

Comment:
I think moving this to issues is the key thing here and "exclude" makes more sense than "skip" in that context!

I wonder if "exclude" is the best word to use here? It still hints that something might not be run or processed. What do you feel about the term "ignore", `ignore-dirs` and `ignore-files`? I think for me that clearer signals that they will be processed and added as issues but they will be ignored. This is also in line with the documentation saying that issues won't be reported meaning the issues will be there (not excluded) but not reported (ignored).

Reply:
the term "ignore" is also ambiguous and in the future, it can be more confusing: https://github.com/golang/go/issues/42965#issuecomment-1974089649

the term "exclude" is already used for the other options in this section, so it seems more appropriate.